compiler/vmdef.nim

```diff
@@ -261,11 +261,23 @@ type
     config*: ConfigRef
     graph*: ModuleGraph
     oldErrorCount*: int
+    profile*: Profile
+
+  FileLine* = object
+    fileIndex*: FileIndex
+    line*: uint16
+
+  Profile* = Table[FileLine, float]
 
   TPosition* = distinct int
 
   PEvalContext* = PCtx
 
+proc hash*(f: FileLine): Hash =
+  result = result !& f.line.int
```

Comment:
this is simpler:
```nim
proc hash*(f: FileLine): Hash =
  hash (f.line, f.fileIndex)
```
but ideally, `hash` would work with object, just like toJson/fromJson

Reply:
Dropped the FileLine alltogether, so now hashing TLineInfo instead